src/sidebar/store/modules/annotations.ts

```diff
@@ -603,6 +603,31 @@ const usersWhoAnnotated = createSelector(
   },
 );
 
+/**
+ * Return the list of unique users who have been mentioned in an annotation.
```

Comment:
```suggestion
 * Return the list of unique users who have been mentioned in currently loaded annotations.
```

Clarify that this returns users mentioned in any annotation, as opposed to a specific annotation.